Components/Rating.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PlayX : Rating</title>
+    <link rel="stylesheet" href="../Styles/Rating.css" />
+    <link rel="stylesheet" href="../Styles/main.css" />
+    <link rel="icon" href="../Assets/Screenshot 2022-02-12 222538.png" type="image/png" />
+    <script src="https://kit.fontawesome.com/fa50d5d333.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav class="nav-containerOne flex  p-m">
+        <div class="nav-brand font-xl f-l"><a class="nav-icon-left" href="../index.html">PlayX</a></div>
+        <ul class=" flex flex-center ">
+            <li class="pr-l f-m"><a href="https://github.com/aniketsingh13" class="nav-icon" target="blank"><i
+                        class="fab fa-github"></i></a></li>
+            <li class="pr-l f-m"><a href="https://twitter.com/ANIKETS42266714" class="nav-icon" target="blank"><i
+                        class="fab fa-twitter"></i></a>
+            </li>
+            <li class="pr-l f-m"><a href="https://www.linkedin.com/in/aniket-singh-3332a5198/" class="nav-icon"
+                    target="blank"><i class="fab fa-linkedin"></i></a>
+            </li>
+        </ul>
+    </nav>
+    <main class="flex mt-xl">
+         <!--aside container  -->
+         <aside class="aside__container font-m f-s ">
+            <ul class="">
+                <li class="p-s "><a class="side_logo f-m font-l " href="../Pages/document.html">Getting Started</a></li>
```

Comment:
You can add a classname or remove it, don't leave the class field empty.

Reply:
will change it